Assets/SEE/UI/Window/ConsoleWindow/ConsoleWindow.cs

```diff
@@ -0,0 +1,691 @@
+        {
+            if (channels.TryGetValue(channel, out Channel c))
+            {
+                c.Levels[level] = new(level, color, true);
+                ChannelChanged?.Invoke();
+            }
+            else
+            {
+                Debug.LogWarning($"Channel {channel} doesn't exist");
+            }
+        }
+
+        /// <summary>
+        /// Sets whether 
+        /// </summary>
+        /// <param name="channel"></param>
+        /// <param name="level"></param>
+        /// <param name="enabled"></param>
+        public static void SetChannelLevelEnabled(string channel, string level, bool enabled)
+        {
+            if (channels.TryGetValue(channel, out Channel c)) {
+                if (c.Levels.TryGetValue(level, out ChannelLevel l))
+                {
+                    l.enabled = enabled;
+                    ChannelChanged?.Invoke();
+                } else
+                {
+                    Debug.LogWarning($"Level {channel} doesn't exist for channel {channel}.");
+                }
+            } else
+            {
+                Debug.LogWarning($"Channel {channel} doesn't exist.");
+            }
+        }
+
+
+        /// <summary>
+        /// Returns the message index to which a new message should be appended.
+        /// -1 if the a new message should be created.
+        /// </summary>
+        /// <param name="channel">The channel.</param>
+        /// <param name="level">The level.</param>
+        /// <returns></returns>
+        private static int AppendTo(string channel, string level)
+        {
+            for (int i=messages.Count-1; i>=0; i--)
+            {
+                Message message = messages[i];
+                if (message.Channel == channel && message.Level == level)
+                {
+                    if (!message.Text.EndsWith('\n'))
+                    {
+                        return i;
+                    }
+                }
+            }
+            return -1;
+        }
+
+        /// <summary>
+        /// Initializes the component.
+        /// </summary>
+        protected override void Start()
+        {
+            Title ??= "Console";
+            base.Start();
+            MessageAdded += OnMessageAdded;
+            MessageChanged += OnMessageChanged;
+            MessagesCleared += OnMessagesCleared;
+            ChannelChanged += OnChannelChanged;
+        }
+
+        /// <summary>
+        /// Initializes the component for the desktop platform.
+        /// </summary>
+        protected override void StartDesktop()
+        {
+            base.StartDesktop();
+            Transform root = PrefabInstantiator.InstantiatePrefab(windowPrefab, Window.transform.Find("Content"), false).transform;
+            items = (RectTransform)root.Find("Content/Items");
+            foreach (Transform child in items)
+            {
+                Destroyer.Destroy(child.gameObject);
```

Comment:
Why is this part necessary?

Reply:
The console window has some dummy messages.
So the window isn't empty when looking at it in Unity / in the prefab editor.

Reply:
Ah, thanks. Could you add that as a comment here?

Reply:
Sure.